operator/pom.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.streamshub</groupId>
+        <artifactId>console</artifactId>
+        <version>0.1.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>console-operator</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <!-- Container image properties for systemtests -->
+        <container-image.registry>localhost</container-image.registry>
+        <container-image.group>streamshub</container-image.group>
+        <container-image.tag>${project.version}</container-image.tag>
+        <container-image.push>false</container-image.push>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
```

Comment:
Any particular reason you're using the "internal" BOM as opposed to the platform one?

Reply:
For some reference: https://developers.redhat.com/articles/2023/09/19/write-operators-java-josdk-part-4-upgrading-strategies#strategies_for_qosdk_and_quarkus_updates

Reply:
I've updated this and the plugin references to use `io.quarkus.platform` using properties. Thanks